### PR TITLE
New bugs

### PR DESCRIPTION
**Pull request by encukou**
Remember the time I nerd-sniped myself on portingdb?
This is probably the last usable thing from that.

When portingdb started, it collected "links" for upstream URL, the py3 porting bug, Bugzillas, etc.
These are now usually unused – we don't want to store information in portingdb if we can avoid it.

This pulls bugs out of links, makes it possible to add more of them, tracks the trackers, and displays them on the package page:

image

### CODE REVIEW

**Comment by hroncok** [comment]
This look useful, I recently thought: maybe we need to track the removal bugs here, but not mark packages mispackaged on them. Will check the code shortly!

File: dnf-plugins/py3query.py [hunk 1]
```diff
@@ -30,22 +30,17 @@
...
-]
-# Trackers of bugs whose presence gives us additional information about the
-# status quo.
-ADDITIONAL_TRACKER_BUGS = [
-    1312032,  # PY3PATCH-AVAILABLE
-    1333770,  # PY3PATCH-PUSH
-    1432186,  # Missing PY3-EXECUTABLES
-]
+TRACKER_BUGS = {
+    1698500: "F31_PY2REMOVAL",
```
Comment (hroncok) [inline dnf-plugins/py3query.py]: and PY2REMOVAL, sadly.

Comment (hroncok) [inline dnf-plugins/py3query.py]:
```suggestion
    1625773: "PY2REMOVAL",
    1698500: "F31_PY2REMOVAL",
```

**Comment by encukou** [comment]
Thanks! I already had a commit for adding it.
The data update is in #574.

**Comment by encukou** [comment]
Merged via #574

**Pull request closed without merging**